apps/fabric-website-resources/src/components/DemoPage.tsx

```diff
@@ -49,6 +49,7 @@ export const DemoPage: React.StatelessComponent<IDemoPageProps> = componentPageP
       componentStatus={
         componentPageProps.componentStatus ? <ComponentStatus {...componentPageProps.componentStatus} /> : undefined
       }
+      feedback={componentPageProps.feedback ? <PageMarkdown>{componentPageProps.feedback}</PageMarkdown> : undefined}
```

Comment:
you can probably do `componentPageProps.feedback && <PageMarkdown>{componentPageProps.feedback}</PageMarkdown>`

Reply:
For some reason, the ComponentPage doesn't allow for null... must be "undefined"